replicated-security/noble-1/noble-1-genesis-without-ccv.json

```diff
@@ -0,0 +1,233 @@
+{
+    "genesis_time": "2023-02-02T17:34:18.048979179Z",
+    "chain_id": "noble-1",
+    "initial_height": "1",
+    "consensus_params": {
+      "block": {
+        "max_bytes": "22020096",
+        "max_gas": "-1",
+        "time_iota_ms": "1000"
+      },
+      "evidence": {
+        "max_age_num_blocks": "100000",
+        "max_age_duration": "172800000000000",
+        "max_bytes": "1048576"
+      },
+      "validator": {
+        "pub_key_types": ["ed25519"]
+      },
+      "version": {}
+    },
+    "app_hash": "",
+    "app_state": {
+      "auth": {
+        "params": {
+          "max_memo_characters": "256",
+          "tx_sig_limit": "7",
+          "tx_size_cost_per_byte": "10",
+          "sig_verify_cost_ed25519": "590",
+          "sig_verify_cost_secp256k1": "1000"
+        },
+        "accounts": [
+            {
+                "@type": "/cosmos.auth.v1beta1.BaseAccount",
+                "account_number": "0",
+                "address": "noble1lwklzs6ejglapp6eccxtfmajwykt8nrdh8fjp8",
+                "pub_key": null,
+                "sequence": "0"
+            },
+            {
+                "@type": "/cosmos.auth.v1beta1.BaseAccount",
+                "account_number": "0",
+                "address": "noble1hrmxjjw7egcccvgrwq6seh2x3wh5curew9d268",
+                "pub_key": null,
+                "sequence": "0"
+            },
+            {
+                "@type": "/cosmos.auth.v1beta1.BaseAccount",
+                "account_number": "0",
+                "address": "noble17krs6fx6vtzpampnemmdmuw5y8m2thmtsyxk90",
+                "pub_key": null,
+                "sequence": "0"
+            },
+            {
+                "@type": "/cosmos.auth.v1beta1.BaseAccount",
+                "account_number": "0",
+                "address": "noble17ln5m096wnj9dtvj3wyj0xl98yz53ptkaj7u66",
+                "pub_key": null,
+                "sequence": "0"
+            },
+            {
+                "@type": "/cosmos.auth.v1beta1.BaseAccount",
+                "account_number": "0",
+                "address": "noble13fzugw54k8h3qne8v7v5ex995rejzv8h8v9fu4",
+                "pub_key": null,
+                "sequence": "0"
+            }
+        ]
+      },
+      "authz": {
+        "authorization": []
+      },
+      "bank": {
+        "params": {
+          "send_enabled": [],
+          "default_send_enabled": true
+        },
+        "balances": [],
```

Comment:
Will you be able to set up a relayer if there are no funded accounts for gas fees?

Reply:
Gas fees will be 0 at launch